### DISCUSSION TITLE
No way to filter by directory

### DISCUSSION THREAD

**Discussion by alex8bitw** [tj-actions/changed-files#2276] [Q&A] [answered]
Hi,
  I opened a bug, which was immediately closed, regarding not being able to filter by directory:  https://github.com/tj-actions/changed-files/issues/2273
  `files:` returns nothing, while `dir_names_include_files:` returns everything.

There doesn't seem a way to filter just on directories.

**Comment by jackton1** [comment, marked as answer]
HI @alex8bitw, this action only supports filtering changed files, hence the name `changed-files`. 

To match a directory that contains a changed file, you'll need to use a directory pattern. 

### Example
Match directories with the word `hono` in them
```
**/*hono*/**
```

> \[!NOTE]
> The use of `/**` to match files within the folder and `dir_names ` set to `true` essentially returns the directory name of the matched changed file.
